FreeRTOS/FreeRTOS-Plus-TCP
# IPv4/single: updating NetworkInterface sources

Description
-----------
Some of the latest updates of the NetworkInterface sources:

- Updated copy right notices and release versions (V2.3.4)
- DriverSAM: variable `uxCurrentSemCount` was not defined
- STM32Fxx : added support for STM32F1 and STM32F2
- phyHandling : added support for 2 more PHY's (DP83TC811S and MV88E6071).
- Removed definition of `ipFRAGMENT_OFFSET_BIT_MASK` from STM32Fxx/NetworkInterface.c

Test Steps
-----------
The changes were tested on an STM32F4x and a on a SAME70 board.
Also users [on the FreeRTOS forum](https://forums.freertos.org/t/i-have-heavy-ping-lose-rate-on-my-stm32f107-board-with-freertos-tcp/11178) helped with testing.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
